src/app_service/service/token/service_items.nim

```diff
@@ -0,0 +1,93 @@
+import strformat, Tables
+
+import app_service/common/types as common_types
+#TODO: remove dependency
+import ../wallet_account/dto/token_dto
+
+# This file hold the data types used by models internally
+
+type SupportedSourcesItem* = ref object of RootObj
+    name*: string
+    updatedAt* : int64
+    source*: string
+    version*: string
+    # Needed to show upfront on ui count of tokens on each list
+    tokensCount*: int
+
+proc `$`*(self: SupportedSourcesItem): string =
+  result = fmt"""SupportedSourcesItem[
+    name: {self.name},
+    updatedAt: {self.updatedAt},
+    source: {self.source},
+    version: {self.version},
+    tokensCount: {self.tokensCount}
+    ]"""
+
+type
+  TokenItem* = ref object of RootObj
+    # key is created using chainId and Address
+    key*: string
+    name*: string
+    symbol*: string
+    # uniswap/status/custom seq[string]
+    sources*: seq[string]
+    chainID*: int
+    address*: string
+    decimals*: int
+    # will remain empty until backend provides us this data
+    image*: string
+    tokenType*: common_types.NewTokenType
```

Comment:
Just a suggestion: Since it's part of a `TokenItem` object definition, it seems to me redundant to add `token` word!
```suggestion
    type*: common_types.NewTokenType
```